docs/internal/proposals/error-handling/error-handling.md

```diff
@@ -0,0 +1,48 @@
+# Error handling in Director graphql
+
+In compass director, we have to somehow filter errors which should be visible to user.
+Basic list of errors which should be displayed to the user:
+* Internal error
+* Invalid Data
+* Not Unique
+* Not Found
+* Tenant Not Found
+* Insufficient Scopes
+
+This list can be split into 2 groups due to different handling:
+* internal errors
+* the rest of errors, which will be presented to the user
+
+In group of internal errors, are errors from:
+* external libraries
+* panics
+* postgreSQL, except errors like NotFound, NotUnique
+
+To deal with the errors we introduced custom errors, which contains error codes.
+
+## Proposed custom errors and theirs error codes (the error codes can change):
+
+| Error type           | Error code  |                            Description                                                      |
```

Comment:
Why are you mixing two naming conventions here? One with space in between and one without space. I think we should have only one convention.